cmd/go-qcow2reader-example/read.go

```diff
@@ -67,7 +67,19 @@ func cmdRead(args []string) error {
 
 	buf := make([]byte, bufferSize)
 	sr := io.NewSectionReader(img, offset, length)
-	_, err = io.CopyBuffer(os.Stdout, sr, buf)
+	w := &Writer{os.Stdout}
+
+	_, err = io.CopyBuffer(w, sr, buf)
 
 	return err
 }
+
+// Writer forces os.File to write using our buffer by hiding it's ReadFrom
```

Comment:
it's → its ?

Reply:
Not sure, too easy to get this wrong :-)

Reply:
Right, should be "its"

Reply:
Removed in current version.